app/src/main/java/com/farmerbb/taskbar/util/U.java

```diff
@@ -713,110 +730,14 @@ public static void setCachedRotation(int cachedRotation) {
 
     public static String getTaskbarPosition(Context context) {
```

Comment:
Actually, should we go ahead and move this whole method over to `TaskbarPosition` as well to keep everything in one place?

Reply:
No problem. I will add a new commit to do it in this `PR`.